src/hooks/useActionCenterSubscribers/useThorchainLpWithdrawActionSubscriber.tsx

```diff
@@ -0,0 +1,112 @@
+import { fromAccountId } from '@shapeshiftoss/caip'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { useCallback, useEffect } from 'react'
+
+import { useNotificationToast } from '../useNotificationToast'
+
+import { useActionCenterContext } from '@/components/Layout/Header/ActionCenter/ActionCenterContext'
+import { GenericTransactionNotification } from '@/components/Layout/Header/ActionCenter/components/Notifications/GenericTransactionNotification'
+import { waitForThorchainUpdate } from '@/lib/utils/thorchain'
+import { actionSlice } from '@/state/slices/actionSlice/actionSlice'
+import { selectPendingThorchainLpWithdrawActions } from '@/state/slices/actionSlice/selectors'
+import type { GenericTransactionAction } from '@/state/slices/actionSlice/types'
+import {
+  ActionStatus,
+  ActionType,
+  GenericTransactionDisplayType,
+  isGenericTransactionAction,
+} from '@/state/slices/actionSlice/types'
+import { selectTxs } from '@/state/slices/selectors'
+import { serializeTxIndex } from '@/state/slices/txHistorySlice/utils'
+import { useAppDispatch, useAppSelector } from '@/state/store'
+
+export const useThorchainLpWithdrawActionSubscriber = () => {
+  const dispatch = useAppDispatch()
+  const { openActionCenter, isDrawerOpen } = useActionCenterContext()
+  const toast = useNotificationToast({ duration: isDrawerOpen ? 5000 : null })
+
+  const pendingThorchainLpWithdrawActions = useAppSelector(selectPendingThorchainLpWithdrawActions)
+  const txs = useAppSelector(selectTxs)
+
+  const handleThorchainUpdate = useCallback(
+    async (action: GenericTransactionAction) => {
+      const { txHash } = action.transactionMetadata
+
+      try {
+        // Wait for THORChain to actually process and update the LP position
+        await waitForThorchainUpdate({
```

Comment:
Plan for follow-up is to leverage action status (i.e this guy), at least for withdraw Txs - status introspection is super derped: it only really takes a few seconds for it to be seen and processed (assuming a broadcasted Tx from a fast chain obviously), but since the polling interval is 60s, the first poll in `<TransactionRow />` is too fast, and we don't get to have another poll within 60s. 
Whereas with the lil' interval it takes for the notification center subscribers to kick in, we get a complete update on the first poll. 

Also, super derp to have two pollers either way! 

But yeah, definitely out of scope.